src/card.ts

```diff
@@ -213,4 +213,38 @@ export class CardApi {
     }
     return { success: !resp?.payload?.errorCode, body: { ...resp.payload } }
   }
+
+  /*
+   * Function
+   */
+  async update(token: string, options: {
+    metadata?: any,
+    expedite?: boolean,
+    fulfillment?: Fulfillment,
+    token?: string,
+    userToken?: string,
+  }): Promise<{
+    success: boolean,
+    body?: Card,
+    error?: MarqetaError,
+  }> {
+    const resp = await this.client.fire('PUT',
+      `cards/${token}`,
+      undefined,
+      options
+    )
+    // catch any errors...
+    if (resp?.payload?.errorCode) {
+      return {
+        success: false,
+        error: {
+          type: 'marqeta',
+          error: resp?.payload?.errorMessage,
+          status: resp?.payload?.errorCode,
+        },
+      }
+    }
+    return { success: !resp?.payload?.errorCode, body: { ...resp.payload } }
+  }
+
```

Comment:
Generally don't leave a blank line, but I suspect this is still going to add more so no big deal... unless it's the last bit of code for this file...